javascript-view-swapping/main.js

```diff
@@ -0,0 +1,30 @@
+var tabContainer = document.querySelector('.tab-container');
+var tabs = document.querySelectorAll('.tab');
+var views = document.querySelectorAll('.view');
+
+function clickHandler(event) {
+  if (event.target.matches('.tab')) {
+    for (var i = 0; i < tabs.length; i++) {
+      if (tabs[i] === event.target) {
+        tabs[i].className = 'tab active';
+      } else {
+        tabs[i].className = 'tab';
+      }
+    }
+  }
+  if (event.target.className !== 'tab active') {
+    return;
+  }
```

Comment:
Is this doing anything? After the above loop, the `event.target` should definitely have the class list `"tab active"`